packages/kolibri/composables/useNav.js

```diff
@@ -65,9 +67,13 @@ function validateNavItem(component) {
 }
 
 export const registerNavItem = component => {
-  if (!navItems.includes(component)) {
+  if (!i18nReady.value) {
```

Comment:
All the changes in this file were necessitated by the fact that the nav item registration process validates that the nav items have a label - which is invariably a translated string. All the changes here do is wait for i18nReady to be true, and then do the validation.

An alternative here would be to avoid the complexity and instead change the validation to avoid the string being evaluated early.